.chloggen/gcp_genai_dynamic_req_attr.yaml

```
@@ -0,0 +1,5 @@
+change_type: enhancement
+component: [gcp, gen-ai]
```

Can you update as below? I created a PR want to handle multiple components case at https://github.com/open-telemetry/semantic-conventions/pull/2209
```suggestion
component: gcp, gen-ai
```

---

`[]` would be more idiomatic way to represent multiple values in yaml, why limit us to concatenated strings?

---

Yes, but seems we are not following best practise here, like with one value, we are using `component: val1`, for multiple values, we are using `component: [val1, val2]`, we should probably go with `component: [val1]` even if there is only one value.

I have updated https://github.com/open-telemetry/semantic-conventions/pull/2209, it can handle following cases now, we should be good. 
- `component: val1`
- `component: [val1, val2]`
- `component: [val1]`